### ISSUE TITLE
DRAEM — Baseline using MVTec training and custom testing

### ISSUE THREAD

**Issue by Tiff223** [Tiff223/Leather_inspection#23]
## Purpose

Measure how the frozen MVTec-trained DRAEM model and its source-domain post-processing transfer to the custom leather domain.

This issue is one track of the model's three-track leather experiment. Benchmark, custom-trained, and MVTec-to-custom transfer results must remain separate and must never be averaged.

## Model-specific recording requirements

Record anomaly-source dataset and provenance, synthetic-anomaly settings, SSPCAB setting, beta range, input size, epochs, optimizer settings, device, and all threshold/post-processing values. Synthetic anomalies must not use validation or test images.

## Pipeline preparation

Before execution:

- [ ] Create the MVTec and custom configuration files in the appropriate `configs/mvtec/` and `configs/custom_leather/` folders
- [ ] Implement and verify every required evaluator and report adapter
- [ ] Register `draem` in `pipeline/leather_pipeline.json`
- [ ] Confirm every stage is `READY`; do not bypass a `NOT WIRED` stage
- [ ] Run dataset preflight and confirm no cross-split byte duplicates
- [ ] Preview the full master pipeline

Master pipeline preview:

```bash
python pythons/leather_three_track_pipeline.py plan --model draem \
  --track all --run-id draem_three_track_run01
```

Master pipeline execution:

```bash
python pythons/leather_three_track_pipeline.py run --model draem \
  --track all --run-id draem_three_track_run01 --execute
```

If execution stops, fix the cause and repeat the exact command with the same master run ID. Completed stages must be skipped and preserved.

## Dataset protocol

- Source fitting: MVTec AD leather only
- Custom training images used: 0
- Calibration: independent 31-image custom validation set
- Evaluation: 33-image legacy common custom test
- Do not retrain or fine-tune the representation on custom images
- Run this track only after the MVTec benchmark checkpoint is complete

## Required operating points

1. **Strict frozen transfer:** preserve the MVTec checkpoint, normalization/calibration values, anomaly-map statistics, and MVTec threshold.
2. **Validation-calibrated transfer:** keep the representation frozen and adapt only explicitly permitted post-processing using independent custom validation.

Never replace the strict result with the adapted result.

## Track checklist

- [ ] Confirm the exact MVTec checkpoint and preserved source calibration values
- [ ] Preview and execute `transfer_baseline`
- [ ] Verify master/child ledgers and SHA-256 dataset manifest
- [ ] Verify zero custom training images entered fitting
- [ ] Record validation ranking, chosen adapted threshold, metrics, mistakes, and score overlap
- [ ] Record strict-transfer test AUROC/AP and threshold-dependent metrics
- [ ] Record validation-adapted test metrics separately
- [ ] Generate per-image scores and mistake heat maps
- [ ] Preview and execute `transfer_report`
- [ ] Generate and integrity-check the DOCX report
- [ ] Execute the transfer `update_comparison`
- [ ] Verify both operating points appear only in the transfer/domain-shift section
- [ ] Verify the custom-trained ranking and score-range table were not changed by transfer data

## Artifacts and results

- Source benchmark issue/checkpoint:
- Master run ID: `draem_three_track_run01`
- Native MVTec threshold/calibration:
- Validation-adapted threshold:
- Dataset manifest:
- Stage ledgers:
- Validation/test summaries:
- Per-image scores and figures:
- Report:
- Transfer comparison files:

## Limitations

Explain domain shift, distinguish threshold-independent ranking from threshold-dependent classification, and identify the legacy custom test as previously examined. Pixel metrics require matching custom masks.

## Conclusion

Not completed yet. Close only after the local artifacts and transfer-only comparison placement have been reviewed.
